frontend/src/app/search/loading.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+// Defines a loading component for the search feature
+export default function Loading() {
+  const skeletonStyle = {
+    backgroundColor: "#eee",
+    borderRadius: "4px",
+    height: "20px",
+    margin: "10px 0",
+    width: "50%",
+  };
+
+  return (
+    <ul>
+      {Array.from({ length: 25 }).map((_, index) => (
+        <li key={index} style={skeletonStyle} />
+      ))}
+    </ul>
+  );
+}
```

Comment:
@andycochran we probably don't need a skeleton loader for an MVP - but it's pretty straightforward to setup here. We can probably turn the css into some css classes at some point so things are more organized